domains/faultysden.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "FaultyFaulty-glitch",
+           "email": "faultywindows+github@gmail.com",
+           "discord": "1041028534690386051"
+        },
+    
+        "record": {
+            "TXT": "google-site-verification=ix1jczu2q87igryc4eaexvdv1vrzjxolu14sjfpsnko"
```

Comment:
TXT record alone doesn't do anything here.

Reply:
Well Google Search Console told me to put that alone.

Reply:
I'm just my other github account, but the thing is Google Search Console told me that this is the only thing I need to do

Reply:
why do you need to add a website to google without anything in it

Reply:
Bro listen, I want to add an custom domain to my Google Sites page.
It said my domain is not verified, so I need to verifiy it with that txt record.

Reply:
Can't you verify your Google Sites page via HTML or did I get that information elsewhere?

Reply:
> Can't you verify your Google Sites page via HTML or did I get that information elsewhere?

Yeah, but where would I put it?